src/openMVG/matching_image_collection/GPU_Matcher_Regions.hpp

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
header `
// This file is part of OpenMVG, an Open Multiple View Geometry C++ library.`